resources/manifests/grafana_values.yaml

```diff
@@ -16,6 +16,9 @@ grafana.ini:
   auth:
     disable_login_form: true
     disable_signout_menu: true
+  server:
+    # this is required to use Grafana behind a reverse proxy (caddy)
+    root_url: "%(protocol)s://%(domain)s:%(http_port)s/grafana/"
```

Comment:
just for my own understanding, grafana stuff is in `manifests/` for legacy reasons, yeah? was surprised its not also being deployed by helm (not a blocker for this PR tho)